src/pages/components/menu.mdx

```diff
@@ -169,48 +200,76 @@ import MenuSizeCode from '!!raw-loader!../../examples/dropdowns/menu/Size.tsx';
 
 <Component components={props.data.mdx.components} componentName="item" />
 
+An Item is used for each dropdown option. It must be nested within a Menu component. It requires a `value` prop,
```

Comment:
What do you think about using the term "item" instead of "option"? 

Confusion regarding menu vs select dropdowns is common, so I think Garden can use words that better distinguish the two. Menu dropdown items have `role="menuitem"`. This is different for select dropdowns items which have `role="option"`.

Reply:
Great point. I had resorted to it because of how "An Item is used for each dropdown item" sounded. I'll try to rephrase it.

Reply:
Fair point. I removed the part about an item being an item. 😅